.github/config.yml

```diff
@@ -17,8 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-istio/master/.github/welcome/Layer5-celebration.png)
-
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-darkmode-celebration.svg">
```

Comment:
@duckling69  Please have a look at the path <source media="(prefers-color-scheme: dark)"
srcset="https://raw.githubusercontent.com/meshery/meshery-istio/master/.github/welcome/Meshery-darkmode-celebration.svg">